src/Raven.Server/NotificationCenter/NotificationCenterWebsocketWriter.cs

```diff
@@ -56,18 +55,6 @@ public async Task WriteNotifications(Func<string, bool> shouldWriteByDb)
                             continue;
                         }
 
-                        if (shouldWriteByDb != null &&
-                            shouldWriteByDb((string)tuple.Item2["Database"]) == false)
-                        {
-                            if (sp.ElapsedMilliseconds > 5000)
-                            {
-                                sp.Restart();
-                                await SendHeartbeat();
```

Comment:
We actually probably need this here. If we have a lot of events / messages, but none that we can send, we need to do the heartbeat here.

Reply:
see note above - when we return null such message is not queued, so we timeout on queue.get and heartbird is sent

Reply:
here: https://github.com/ravendb/ravendb/pull/8837/files/a22b9775326dfcd390fe541d5d948d731f9990ac#diff-88c47ac394f9600c53afb59060c78863L52

Reply:
Okay, looks good, then